### DISCUSSION TITLE
Welcome to AndroidGithubSearch Discussions!

### DISCUSSION THREAD

**Discussion by LeoAndo** [LeoAndo/AndroidGithubSearch#99] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 123-ryu** [comment]
aiを学習している宇田川と言うものです。
サイト ディ-プラ-ニング入門chainerチュ-トリアルの14.chainerの基礎の[14]：の終わりの方に表が有ります。
中のepoch 29iteration150とあるが一般的にはiteration300としなければならず何故150で終わっているのか納得ないか理由を見つけたいのですが返信を待っています。
